packages/flame_svg/lib/svg.dart

```diff
@@ -55,11 +55,24 @@ class Svg {
     Vector2 size, {
     Paint? overridePaint,
   }) {
-    final localSize = size.toSize();
+    // Scale the canvas to the size of the destination clip bounds
+    // This is necessary to avoid blurriness when having a
+    // camera.viewfinder.zoom larger than 1.0
+    final widthRatio = canvas.getDestinationClipBounds().size.width /
+        canvas.getLocalClipBounds().size.width;
+    final heightRatio = canvas.getDestinationClipBounds().size.height /
+        canvas.getLocalClipBounds().size.height;
+
+    final localSize = Size(size.x * widthRatio, size.y * heightRatio);
```

Comment:
What happens if the width and height ratio aren't the same?

Reply:
A bug was discovered when trying this out, addressed [here](https://github.com/flame-engine/flame/pull/3616#issuecomment-2947264181)